sdk/include/reactos/x86x64/Intel/Cpuid.h

```diff
@@ -2077,21 +2081,44 @@ typedef union {
     ///
     /// [Bits 7:0] Used for XCR0.
     ///
-    UINT32    XCR0      : 1;
+    UINT32    XCR0      : 8;
     ///
-    /// [Bit 8] PT STate.
+    /// [Bit 8] PT state.
     ///
     UINT32    PT        : 1;
     ///
     /// [Bit 9] Used for XCR0.
     ///
     UINT32    XCR0_1    : 1;
-    UINT32    Reserved1 : 3;
     ///
-    /// [Bit 13] HWP state.
+    /// [Bit 10] PASID state.
+    ///
+    UINT32    PASID     : 1;
+    ///
+    /// [Bit 11] CET user state.
+    ///
+    UINT32    CET_U     : 1;
+    ///
+    /// [Bit 12] CET supervisor state.
+    ///
+    UINT32    CET_S     : 1;
+    ///
+    /// [Bit 13] HDC state.
+    ///
+    UINT32    HDC       : 1;
+    ///
+    /// [Bit 14] UINTR state.
+    ///
+    UINT32    UINTR     : 1;
+    ///
+    /// [Bit 15] LBR state.
+    ///
+    UINT32    LBR       : 1;
+    ///
+    /// [Bit 16] HWP state.
     ///
-    UINT32    HWPState  : 1;
-    UINT32    Reserved8 : 18;
+    UINT32    HWP       : 1;
```

Comment:
Question: are these your own corrections to the header, or was this header actually synced from a 3rd-party source? (the years in the copyright notice at the top of the file stayed unchanged, hence my question.)